core/api/src/graphql/public/types/payload/ln-invoice.ts

```diff
@@ -1,6 +1,6 @@
 import IError from "../../../shared/types/abstract/error"
 
-import LnInvoice from "../object/ln-invoice"
+import LnInvoice from "../../../shared/types/object/ln-invoice"
```

Comment:
shared object should be in /shared/.

this may mean that `core/api/src/graphql/public/types/payload/ln-invoice.ts` may need to be in `core/api/src/graphql/shared/types/payload/ln-invoice.ts` if it imports an object from shared (maybe, not sure)

Reply:
seems they have been renamed already.